### PR TITLE
Added link to awesome-complexity list

### PR DESCRIPTION
**Pull request by sellisd**
https://github.com/sellisd/awesome-complexity

A manually curated list of resources on Complex Systems Science that contains the most important and relevant resources (research, outreach and educational) relevant to Complex Systems Science. Some of its entries might be found in other awesome lists but currently there is no other single curated list that brings together resources on Complex Systems.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by arw36** [comment]
Small edits per the formatting guidelines: 

- Table of Contents section should be named Contents
-  Make sure to end all entries with a period

**Comment by sellisd** [comment]
Thanks @arw36 for the feedback. I just committed your suggestions (commit 0832ae758a22a6a79abc821894e0db16a5786ff8)

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/sellisd/awesome-complexity
✖ Linting

  README.md:105:3
  ✖    1:1    Missing License section                                 remark-lint:awesome/license
  ✖    1:1    Missing or invalid Table of Contents                    remark-lint:awesome/toc
  ✖    4:11   Text "github" should be written as "GitHub"             remark-lint:awesome/spell-check
  ✖    6:10   Text "github" should be written as "GitHub"             remark-lint:awesome/spell-check
  ✖   14:3    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   15:3    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   16:3    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   17:3    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   18:3    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   19:3    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   20:3    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   21:5    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   22:5    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   23:3    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   24:3    Invalid list item link URL                              remark-lint:awesome/list-item
  ✖   45:57   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖   80:15   Remove empty section: "Conferences"                     remark-lint:no-empty-sections
  ✖   84:6    Remove 1 space before this heading’s content            remark-lint:no-heading-content-indent
  ✖  105:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  106:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  107:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  108:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  109:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  110:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  110:87   Found reference to undefined definition                 remark-lint:no-undefined-references
  ✖  111:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  111:121  Found reference to undefined definition                 remark-lint:no-undefined-references
  ✖  112:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  113:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  114:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  115:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  116:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent
  ✖  117:3    Incorrect indentation before bullet: remove 1 space     remark-lint:list-item-bullet-indent

  33 errors
```

https://github.com/sindresorhus/awesome-lint

File: readme.md [hunk 1]
```diff
@@ -306,6 +306,7 @@
 - [Competitive Programming](https://github.com/lnishan/awesome-competitive-programming#readme)
 - [Math](https://github.com/rossant/awesome-math#readme)
 - [Recursion Schemes](https://github.com/passy/awesome-recursion-schemes#readme) - Traversing nested data structures.
+- [Complex Systems science](https://github.com/sellisd/awesome-complexity#readme)
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Comment by sellisd** [comment]
> Closing for lack of response.

Sorry for the lack of response, haven't the time to work on the complexity list it lately. Once I am back on it I will should I do a new pull request or it is possible to continue on this branch?

**Pull request closed without merging**